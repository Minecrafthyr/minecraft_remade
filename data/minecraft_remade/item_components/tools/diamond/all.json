{
  "targets": "#minecraft_remade:tools/diamond",
  "components": {
   "minecraft:max_damage": 1024
  }
}