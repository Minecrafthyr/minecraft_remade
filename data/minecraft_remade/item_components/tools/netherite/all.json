{
  "targets": "#minecraft_remade:tools/netherite",
  "components": {
   "minecraft:max_damage": 2048
  }
}