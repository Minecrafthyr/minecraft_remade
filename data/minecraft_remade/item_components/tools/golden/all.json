{
  "targets": "#minecraft_remade:tools/golden",
  "components": {
   "minecraft:max_damage": 16
  }
}